pkg/volume/cinder/attacher_test.go

```diff
@@ -734,10 +734,6 @@ func (instances *instances) InstanceShutdownByProviderID(ctx context.Context, pr
 	return false, errors.New("unimplemented")
 }
 
-func (instances *instances) InstanceMetadataByProviderID(ctx context.Context, providerID string) (*cloudprovider.InstanceMetadata, error) {
```

Comment:
I wouldn't personally remove the implementations - they will follow the v2 interface as soon as this is introduced then.

I would just remove the changes to cloud.go in the interface.

Reply:
Done, PTAL.

Reply:
yeah agreed with @wojtek-t on this